.github/workflows/build.yml

```diff
@@ -32,18 +44,8 @@ jobs:
         with:
           go-version: ${{ env.GO_VERSION }}
 
-      - uses: technote-space/get-diff-action@v6
-        with:
-          PATTERNS: |
-            **/*.go
-            "!test/"
-            go.mod
-            go.sum
-            Makefile
-
       - name: install
         run: GOOS=${{ matrix.goos }} GOARCH=${{ matrix.goarch }} make build
-        if: "env.GIT_DIFF != ''"
```

Comment:
I'm just trying to understand the implication of removing this. Does it mean that the install will run all the time now even if the diff is null ? If so, isn't this will incur additional costs ? 

But at the same time looks like this is triggered only when changes are made (pull_request or push) so I think these `env.GIT_DIFF` were not very helpful.

Reply:
`env.GIT_DIFF` comes from https://github.com/technote-space/get-diff-action#usage. It was how the action used to work - computing `GIT_DIFF` and requiring this `if` condition to skip the given action.

The install will run only when the changes are made in `paths` (pull_request or push).